docs/GraFx-Studio/guides/template-variables/number/index.md

```diff
@@ -0,0 +1,117 @@
+# Number Variables
+
+## Create a Number Variable
+
+Under the Automate icon, click Variables, and add a variable with the "+" sign.
+
+![screenshot](num0.png)
+
+Choose "Number" as the Variable Type.
+
+![screenshot-full](create-numbers.gif)
+
+## Set the Variable Name
+
+Double-click the name, or choose "Rename" under the "..." menu.
+
+![screenshot](num2.png)
+
+## Set General Properties
+
+![screenshot-full](num1.png)
+
+### Visible
+
+Defines if the variable will be visible in the [Studio UI](/GraFx-Studio/concepts/template-management/#studio-ui) to the end user.
+
+### Range
+
+Defines the acceptable range of values to be accepted by the variable placeholder.
+
+### Decimal Settings
+
+#### Decimal Places
+
+Set the number of decimal places that will be displayed.
+
+#### Decimal Character Style
+
+![screenshot-full](ds0.png)
+
+You can define a character style that is different from the style defined for the text, to use for the decimal part of the number.
+
+The default Paragraph style for the frame is "Text".
```

Comment:
I would mention that it is an example. Now I first read it as if there is always a default paragraph style, which isn't the case.

Reply:
Made it more clear, and used it as example